app/models/chargeback/report_options.rb

```diff
@@ -0,0 +1,102 @@
+class Chargeback
```

Comment:
I was wondering how this works too. It doesn't seem obvious that you can call `ReportOptions.new_from_h` with this code. Although, it's pretty cool that you can.